### PR TITLE
Update README.md with info about new team members

### LINKED ISSUE #138: Update `README` with the info about new team members
## Description
Add the missing information about team members to `README.md`.

## DoD

- Go to [wiki page](https://github.com/ios-course/ironfoudation-team-project/wiki/About-team#meet-the-team) to get the info about the team members.
- Update the `Authors` section for [`README` file](https://github.com/ios-course/ironfoudation-team-project#authors)

### PR DESCRIPTION
**Pull request by julyakmita**
## Summary
This PR is a part of the work to resolve #138.
Add the missing information about team members to `README.md`.

## Changes
- Update the Authors section for `README.md` file.
- Update `CHANGELOG.md`.

## Checks to complete
- [x] My code follows our style guide.
- [x] I've performed a self-review of my code.
- [x] PR's title has a conventional commit style (short and descriptive).
- [x] PR has assignee, reviewers, milestone, it is properly linked to the project and to the issue.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - Update storage keys and storage names to use with the `KeyValueStorage` - [#109](https://github.com/ios-course/ironfoudation-team-project/pull/109)
 - Create `Preferences` model, enum `Role` - [#111](https://github.com/ios-course/ironfoudation-team-project/pull/111)
 - Move `AvatarView` file to the `ReusableViews` folder - [#136](https://github.com/ios-course/ironfoudation-team-project/pull/136)
+- Update the Authors section for the `README.md` file - [#139](https://github.com/ios-course/ironfoudation-team-project/pull/139)
```
Comment (NadzeyaShpakouskaya) [inline CHANGELOG.md:15]: Does it make sense to move this record under `Changed` section instead of `Added`?

**Review by NadzeyaShpakouskaya — CHANGES_REQUESTED** [review]
I have one point to discuss:

**Review by kirpichonok — APPROVED** [review]
LGTM

**Review by marinkman — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (julyakmita) [reply CHANGELOG.md:15]: Yes, good idea. Fixed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ## [Unreleased]
 
+### Changed
+
+- Update the Authors section for the `README.md` file - [#139](https://github.com/ios-course/ironfoudation-team-project/pull/139)
+
```
Comment (NadzeyaShpakouskaya) [inline CHANGELOG.md:12]: nit: 
move the section  `Changed` below `Added`

**Review by NadzeyaShpakouskaya — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 2 above)
Comment (julyakmita) [reply CHANGELOG.md:12]: Ok, thanks. Fixed it.

**Pull request merged**